contracts/middle-layer/TokenHub.sol

```diff
@@ -173,6 +173,29 @@ contract TokenHub is Config, ReentrancyGuardUpgradeable, IMiddleLayer, ITokenHub
      * @param amount The amount to transfer
      */
     function transferOut(address recipient, uint256 amount) external payable override returns (bool) {
+        (uint8 _channelId, bytes memory _msgBytes, uint256 _relayFee, uint256 _ackRelayFee, ) = _prepareTransferOut(
+            msg.sender,
+            recipient,
+            amount
+        );
+
+        ICrossChain(CROSS_CHAIN).sendSynPackage(_channelId, _msgBytes, _relayFee, _ackRelayFee);
+        return true;
+    }
+
+    function prepareTransferOut(
+        address sender,
+        address recipient,
+        uint256 amount
+    ) external payable onlyMultiMessage returns (uint8, bytes memory, uint256, uint256, address) {
+        return _prepareTransferOut(sender, recipient, amount);
```

Comment:
The amount should be limited to 0.1 BNB here.

Reply:
We need a  monitor on relayer, incase there is a bug here and too many transfer out here.

Reply:
Fix the UT please.

Reply:
fixed